packages/vscode-ui5-language-assistant/test/run-integration-tests.ts

```diff
@@ -0,0 +1,28 @@
+import { resolve, dirname } from "path";
+import { runTests } from "vscode-test";
+import globby from "globby";
+
+async function main(): Promise<void> {
+  try {
+    const pkgJsonPath = resolve(__dirname, "..", "..", "package.json");
```

Comment:
we don't care about the package.json, just the dir name so we can resolve it directly.
> resolve(__dirname, "..", "..");